File: common/src/main/java/org/valkyrienskies/clockwork/content/events/ClockworkClientEvents.java
package org.valkyrienskies.clockwork.content.events;

import net.minecraft.client.Minecraft;
import org.valkyrienskies.clockwork.ClockWorkMod;
import org.valkyrienskies.clockwork.content.contraptions.propellor.stream.PropStream;

import static com.jozufozu.flywheel.backend.Backend.isGameActive;

public class ClockworkClientEvents {
    private static final String ITEM_PREFIX = "item." + ClockWorkMod.MOD_ID;
    private static final String BLOCK_PREFIX = "block." + ClockWorkMod.MOD_ID;

    public static void onTickStart(Minecraft client) {

        PropStream.tickClientPlayerSounds();

    }

    public static void onTick(Minecraft client) {
        if (!isGameActive()) {
        }

//        ClockWorkMod.Client.BLUPER_HANDLER.tick();
//        ClockWorkMod.Client.GRAVITRON_HANDLER.tick();
    }


}

File: common/src/main/kotlin/org/valkyrienskies/clockwork/content/events/ClockworkClientEvents.kt
package org.valkyrienskies.clockwork.content.events

import net.minecraft.client.Minecraft
import org.valkyrienskies.clockwork.ClockworkMod
import org.valkyrienskies.clockwork.content.curiosities.tools.auric.gravitron.GravitronRenderHandler
import org.valkyrienskies.clockwork.util.render.BoltUtil

object ClockworkClientEvents {
    private val ITEM_PREFIX = "item." + ClockworkMod.MOD_ID
    private val BLOCK_PREFIX = "block." + ClockworkMod.MOD_ID
    val GRAVITRON_HANDLER = GravitronRenderHandler()

    fun onTickStart(client: Minecraft?) {
    }

    fun onTick(client: Minecraft?) {
        ClockworkMod.OUTLINER.tickOutlines()
        GRAVITRON_HANDLER.tick();
    }

    fun onRenderTick() {
        BoltUtil.tick()
    }
}
